default_query.py

```diff
@@ -182,3 +182,59 @@ def generate_response_best_performance(self, query, annotated_query):
         r = [(i['inner_hits']['sentences']['hits'], i['_source']['ProQ:'], i['_source']['Full text:']) for i in r]
 
         return self.format_response(r[0], question_type = 'What was PERSON best performance')
+
+
+    def generate_response_good_noun(self, query, annotated_query):
+        payload = {
+            "query": {
+                "query_string": {
+                    "query": query.encode('utf-8'),
+                    "fields": ["Full text:"]
+                }
+            }
+        }
+        payload = {
```

Comment:
Womp early morning, yeah previous is unnecessary. Can you change in GUI and merge if good otherwise? I won't be able to get back to it until like 5